Models/MuZero_torch_agent.py

```diff
@@ -52,6 +53,7 @@ def tft_load_model(self, episode):
         path = f'./Checkpoints/checkpoint_{episode}'
         if os.path.isfile(path):
             self.load_state_dict(torch.load(path))
+            self.eval()
```

Comment:
Can we double check that this line doesn't mess up the trainer?

Reply:
Totally